programs/nodeos/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ target_link_libraries( ${NODE_EXECUTABLE_NAME}
         PRIVATE -Wl,${whole_archive_flag} test_control_plugin        -Wl,${no_whole_archive_flag}
         PRIVATE -Wl,${whole_archive_flag} test_control_api_plugin    -Wl,${no_whole_archive_flag}
         PRIVATE -Wl,${build_id_flag}
-        PRIVATE chain_plugin http_plugin producer_plugin http_client_plugin
+        PRIVATE chain_plugin http_plugin producer_plugin http_client_plugin blockvault_client_plugin
```

Comment:
`blockvault_client_plugin` will need wrapped in the `whole_archive_flag` once removed from main.

Reply:
I tried locally after removing it. Didn't need whole_archive. I think maybe it being REQUIRE() from chain_plugin already gets it picked up. Not that it would really hurt anything being whole_archive'd anways